app/controllers/dashboard/sub_service_requests_controller.rb

```diff
@@ -19,46 +19,60 @@
 # TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 class Dashboard::SubServiceRequestsController < Dashboard::BaseController
-  before_action :find_sub_service_request
-  before_filter :protocol_authorizer,     only: [:update_from_project_study_information]
-  before_filter :authorize_admin,         only: :show
+  before_action :find_sub_service_request,  except: :index
+  before_filter :protocol_authorizer,       only: :update_from_project_study_information
+  before_filter :authorize_admin,           only: :show, unless: :format_js?
```

Comment:
Unless JS because the JS format renders the normal view modal, whereas HTML format is for admin use only